lua/telescope/path.lua

```diff
@@ -1,89 +1,33 @@
 local path = {}
 
+local path_is_deprecated = function(name)
+  error('telescope.path.' .. name .. ' is deprecated. Use plenary.path instead')
+end
+
 -- TODO: Can we use vim.loop for this?
 path.separator = package.config:sub(1, 1)
 path.home = vim.fn.expand("~")
 
-path.make_relative = function(filepath, cwd)
-  if not cwd or not filepath then return filepath end
-
-  if filepath:sub(1, #cwd) == cwd  then
-    local offset =  0
-    -- if  cwd does ends in the os separator, we need to take it off
-    if cwd:sub(#cwd, #cwd) ~= path.separator then
-      offset = 1
-    end
-
-    filepath = filepath:sub(#cwd + 1 + offset, #filepath)
-  end
-
-  return filepath
+path.make_relative = function()
```

Comment:
maybe we should just do. Similar to actions deprecated message
```lua
path.__index = function()
  print('deprecated message')
end
```

Reply:
https://github.com/nvim-telescope/telescope.nvim/blob/2c4efc2f8af0575526f7b5dcc35fba9e0f1cc944/lua/telescope/actions/init.lua#L22-L35

Reply:
Just can just always do a error. You don't need to do the string.finds